tests/utils/test_inference.py

```diff
@@ -275,3 +275,41 @@ def test_export_with_other_logger(model_cls, sample_input):
                                                                    save_object_store=None,
                                                                    sample_input=ANY,
                                                                    transforms=None)
+
+
+class LinModel(nn.Module):
+
+    def __init__(self):
+        super().__init__()
+        self.lin1 = nn.Linear(256, 128)
+        self.lin2 = nn.Linear(128, 256)
+
+    def forward(self, x):
+        x = self.lin1(x)
+        x = self.lin2(x)
+        return x
+
+
+@pytest.mark.parametrize(
+    'model_cls',
+    [
+        (LinModel),
+    ],
+)
+def test_dynamic_quantize(model_cls):
+    model = model_cls()
+    model.eval()
```

Comment:
Is this needed? doesn't `export_for_inference()` implementation call `.eval()` after deep copying the model?

Reply:
It's not needed but also doesn't hurt.

Reply:
Will remove